jujugui/static/gui/src/app/components/spinner/_spinner.scss

```diff
@@ -1,21 +1,72 @@
 @keyframes spinner {
   0% {
-    @include rotate(0deg);
+    transform: rotate(0deg);
   }
   100% {
-    @include rotate(360deg);
+    transform: rotate(360deg);
   }
 }
 
-.spinner_loading {
-    @include animation(spinner 1500ms infinite linear);
-    @include create-border-radius(0.5em);
-    @include create-box-shadow(rgba(0, 0, 51, 0.3) 1.5em 0 0 0, rgba(0, 0, 51, 0.3) 1.1em 1.1em 0 0, rgba(0, 0, 51, 0.3) 0 1.5em 0 0, rgba(0, 0, 51, 0.3) -1.1em 1.1em 0 0, rgba(0, 0, 51, 0.3) -1.5em 0 0 0, rgba(0, 0, 51, 0.3) -1.1em -1.1em 0 0, rgba(0, 0, 51, 0.3) 0 -1.5em 0 0, rgba(0, 0, 51, 0.3) 1.1em -1.1em 0 0);
+.spinner-loading {
```

Comment:
Update the style of the spinner to look a bit closer what I think I've seen on some mockups.